js/index.js

```diff
@@ -1,51 +1,54 @@
 import { weatherIcon } from "./tomorrow_io.js"
+import { get_sensor_data } from "./temperature_humidity_sensor"
 
-function updateSensorReadings(jsonResponse) {
-  const inside_temperature = jsonResponse.inside.temperature.toFixed(0);
-  const inside_humidity = jsonResponse.inside.humidity.toFixed(0);
-  const outside_temperature = jsonResponse.outside.temperature.toFixed(0);
-  const outside_humidity = jsonResponse.outside.humidity.toFixed(0);
+async function setSensorReadings() {
+  const sensor_data = await get_sensor_data();
+  const inside_temperature = (sensor_data.inside.temperature / 10)
+  const inside_humidity = sensor_data.inside.humidity.toFixed(0)
+  const outside_temperature = (sensor_data.outside.temperature / 10)
+  const outside_humidity = sensor_data.outside.humidity.toFixed(0)
 
-  document.getElementById("inside-temperature").innerHTML = to_far(inside_temperature) + "°"
+  document.getElementById("inside-temperature").innerHTML = to_fahrenheit(inside_temperature) + "°"
```

Comment:
At some point it might make sense for it to be configurable whether it shows fahrenheit or celsius, and I'm not sure this is the right place for the conversion. But I think this is fine for now.

Reply:
I agree. I'd like to have a button you push to change from F to C and back, but that's a later enhancement imo. I do think it should happen in the JS.